plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -38,8 +37,12 @@ type Elasticsearch struct {
 	OverwriteTemplate   bool
 	ForceDocumentID     bool `toml:"force_document_id"`
 	MajorReleaseNumber  int
```

Comment:
Can you please also add TOML tags to all other user-configurable options in this struct!?

Reply:
ack